package/raphael.js

```diff
@@ -5512,14 +5512,14 @@
                 that.attr(set);
                 (function(id, that, anim) {
                     setTimeout(function() {
-                        eve("raphael.anim.frame." + id, that, anim);
+                        R.stopEvent !== false && eve("raphael.anim.frame." + id, that, anim);
                     });
                 })(that.id, that, e.anim);
             } else {
                 (function(f, el, a) {
                     setTimeout(function() {
-                        eve("raphael.anim.frame." + el.id, el, a);
-                        eve("raphael.anim.finish." + el.id, el, a);
+                        R.stopEvent !== false && eve("raphael.anim.frame." + el.id, el, a);
+                        R.stopEvent !== false && eve("raphael.anim.finish." + el.id, el, a);
```

Comment:
Have a variable to store this Boolean check to reduce one logical operation per frame.